js/arbeidsrettet-oppfolging/HarAlleredeOppfolgingAlertstripe.js

```diff
@@ -8,7 +8,7 @@ const HarAlleredeOppfolgingAlertstripe = () => {
         <Alertstripe type="suksess" className="alleredeOppfolging blokk">
             <div className="alleredeOppfolging__innhold">
                 <Normaltekst className="alleredeOppfolging__tekst">{getLedetekst('infoside-fo.alertstripe.tekst')}</Normaltekst>
-                <a className="lenke" href="/veientilarbeid">
+                <a className="lenke" href="https://veientilarbeid.nav.no">
```

Comment:
Dette fungerer vel bar for Prod.
Kan bruke miljøvariabelen `NAIS_CLUSTER_NAME` og gi preprod url(`veientilarbeid-q1.dev-sbs.nais.io`) om NAIS_CLUSTER_NAME === dev-sbs/prod-fss , ellers prod-url.

Reply:
prod-fss og?

Reply:
bump

Reply:
Jeg skriv visst feil mente sbs, ikke sbs ja.